tests/wdio.conf.js

```diff
@@ -1,3 +1,26 @@
+require('dotenv').load({silent: true});
```

Comment:
This is required to let developers add their Browserstack user and key to the `.env` file.